.github/workflows/docker/sensei/install_sensei.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+DOWNLOAD_ROOT=/root/downloads
+EXTRACT_ROOT=/root/downloads/extracted
+mkdir -p ${DOWNLOAD_ROOT}
+mkdir -p ${EXTRACT_ROOT}
+
+# Download and install Sensei (VTK + MPI + Python)
+export CMAKE_GENERATOR=Ninja
+
+cd ${DOWNLOAD_ROOT}
+mkdir sensei
+cd sensei
+git clone https://gitlab.kitware.com/sensei/sensei.git &&
+cd sensei &&
+git checkout ${SENSEI_VERSION} &&
+mkdir build &&
+cd build &&
+cmake .. \
+  -D BUILD_TESTING=OFF \
+  -D ENABLE_PYTHON=ON \
+  -D ENABLE_MANDELBROT=OFF \
+  -D ENABLE_OSCILLATORS=OFF \
+  -D ENABLE_VTK_MPI=ON \
+  -D MPI4PY_INCLUDE_DIR=/usr/lib64/python3.9/site-packages/openmpi/mpi4py/include &&
```

Comment:
Potential generic:
`-D MPI4PY_INCLUDE_DIR="$(which python3) -c 'import mpi4py; print(mpi4py.get_include())'"`

Reply:
That is better, however it should not be necessary to specify -DMPI4PY_INCLUDE_DIR manually when configuring SENSEI.  
```
module load mpi/openmpi-x86_64
```
would likely resolve the issue.

Reply:
This container does not have environment modules support, so it is not possible to module load anything. Since this is a container that we maintain, I had control over where software is so a hard coded path was just easier.

Reply:
Sure, but you could also `sudo dnf install environment-modules` I guess this is a fedora'ism with they way they support multiple MPI.

Reply:
Env modules are a bit tricky in non-login shells though. Do-able, but probably not needed for the one hint.

I was just wondering if this line could be more generic :)

Reply:
Yeah, I get it, it's a good thought. the reason why cmake (which is intertnally doing what you're suggesting) can't find mpi4py is that it's not in the pythonpath by default on fedora. installing modules and loading the mpi module would solve that.

I've not seen any issues with environment modules in our other fedora ci images ;-)